configs/components/facter.rb

```diff
@@ -112,7 +112,7 @@
     skip_blkid = 'OFF'
   elsif platform.is_rpm?
     if (platform.is_el? && platform.os_version.to_i >= 6) || (platform.is_sles? && platform.os_version.to_i >= 11) || platform.is_fedora?
-      pkg.build_requires "libblkid-devel"
+      pkg.build_requires "libblkid-devel" unless platform.architecture == "s390x"
```

Comment:
rubocop isn't complaining about this. From what I've just looked up, the differences between using "or" vs. "||" only come into play when doing things like using them in assignment, and as if conditionals they are synonymous. Maybe @mckern can weigh in here?